lg_media/scripts/image_viewer.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, viewports, save_path):
         self.lock = Lock()
 
     def director_translator(self, data):
+        global graphic_opts
+        graphic_opts = {}
```

Comment:
why is this a global and not part of the image viewer class? If you add it to the init above as `self.graphic_opts = {}` and access it like that I believe it should work.

Reply:
It works, tested it on ephq